storybook/src/elements/nuxeo-actions-menu/nuxeo-actions-menu.stories.js

```diff
@@ -3,23 +3,9 @@ import { number } from '@storybook/addon-knobs';
 import { html } from 'lit-html';
 import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-actions-menu';
 import '@nuxeo/nuxeo-ui-elements/actions/nuxeo-link-button';
+import items from '../../lists/icons';
 
-const iconsList = [
-  { icon: 'nuxeo:edit', label: 'Edit' },
-  { icon: 'nuxeo:attachment', label: 'Attachment' },
-  { icon: 'nuxeo:dashboard', label: 'Dashboard' },
-  { icon: 'nuxeo:add', label: 'Add' },
-  { icon: 'nuxeo:delete', label: 'Delete' },
-  { icon: 'nuxeo:download', label: 'Download' },
-  { icon: 'nuxeo:export', label: 'Export' },
-  { icon: 'nuxeo:edit', label: 'Edit' },
-  { icon: 'nuxeo:attachment', label: 'Attachment' },
-  { icon: 'nuxeo:dashboard', label: 'Dashboard' },
-  { icon: 'nuxeo:add', label: 'Add' },
-  { icon: 'nuxeo:delete', label: 'Delete' },
-  { icon: 'nuxeo:download', label: 'Download' },
-  { icon: 'nuxeo:export', label: 'Export' },
-];
+const iconsList = items.find((item) => item.name === 'nuxeo').icons;
```

Comment:
If we store the icons list as a map, for example:
`{
  icons: [],
  nuxeo: [],
  others: [],
}`

We can avoid `find` and we reference the subset directly items['nuxeo'].
On `nuxeo-icons` story we can iterate over items keys on our first ` dom-repeat`.

WDYT? It may be easier to use and it may address @semisse concerns 🙂

Reply:
Agree!

Reply:
yes I like that approach more, I pushed the change take a look